agile-app/src/app/map/page.tsx

```diff
@@ -1,7 +1,6 @@
 "use client"
 
 import {useEffect, useRef, useState} from "react";
-import {fetchMunicipalityData, fetchRegionData} from "@/scripts/geoFetching";
 import {GeoJSON, GeoJSONProps, MapContainer, TileLayer} from 'react-leaflet'
 import 'leaflet/dist/leaflet.css'
 import {Feature, GeoJsonObject, GeoJsonProperties, Geometry} from "geojson";
```

Comment:
remove unused import GeoJsonProperties
```suggestion
import {Feature, GeoJsonObject, GeoJsonProperties, Geometry} from "geojson";
``` -> ```suggestion
import {Feature, GeoJsonObject, Geometry} from "geojson";
```